compiler/rustc_error_messages/locales/en-US/query_system.ftl

```diff
@@ -23,3 +23,5 @@ query_system_cycle_recursive_trait_alias = trait aliases cannot be recursive
 query_system_cycle_which_requires = ...which requires {$desc}...
 
 query_system_query_overflow = queries overflow the depth limit!
+
+query_system_layout_of_depth = Query depth increased by {$depth} when {$desc}!
```

Comment:
```suggestion
query_system_layout_of_depth = query depth increased by {$depth} when {$desc}
```

It is somewhat problematic for the future to include `$desc` in a message, because the output might be set up for another language, which will result in embedded english text.